apps/showcase/src/app/component-replacement/component-replacement.template.html

```diff
@@ -0,0 +1,37 @@
+<h1>Component Replacement</h1>
+<div class="row">
+  <div class="right-nav order-1 order-lg-2 col-12 col-lg-2 sticky-lg-top pt-5 pt-lg-0">
+    <o3r-in-page-nav-pres
+      id="component-replacement-nav"
+      [links]="links$ | async"
+    >
+    </o3r-in-page-nav-pres>
+  </div>
+  <div class="order-2 order-lg-1 col-12 col-lg-10">
+    <h2 id="component-replacement-description">Description</h2>
+    <div>
+      <p>This module provides a way to replace the presentation layer of a component inside an application.</p>
```

Comment:
which module should the user install via the ng-add?
Shouldn't we provide some code as an example like we do in https://amadeusitgroup.github.io/otter/#/configuration ?

Reply:
updated